### DISCUSSION TITLE
Inav 7.0.1 speedybee f405 v3 RPM filter

### DISCUSSION THREAD

**Discussion by AltxVik** [iNavFlight/inav-configurator#1944] [Q&A]
Inav 7.0.1 speedybee f405 v3 the RPM filter setting is not available (the fields are not active). Why is that?. is this a limitation for this board? or lack of software? I think the code for RPM filter settings is not complete for speedybee f405 v3

**Comment by MrD-RC** [comment]
It needs telemetry between the ESC and flight controller.

**Reply by AltxVik** [reply]
It's clear. On another controller - omnibus F405, this menu is available for configuration with the same settings and connections.
but I switched to speedybee...
